src/main/java/org/minions/devfund/marcos/moviedatabase/MovieDatabase.java

```diff
@@ -0,0 +1,129 @@
+package org.minions.devfund.marcos.moviedatabase;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * It is in charged to represent a movie database with it is main attributes.
+ */
+public class MovieDatabase {
+
+    private List<Movie> movieList;
+    private List<Actor> actorList;
+
+    /**
+     * Initializes an instance of {@link MovieDatabase}.
+     */
+    public MovieDatabase() {
+        movieList = new ArrayList<>();
+        actorList = new ArrayList<>();
+    }
+
+    /**
+     * Gets the database movies list.
+     *
+     * @return list of {@link Movie}.
+     */
+    public List<Movie> getMovieList() {
+        return movieList;
+    }
+
+    /**
+     * Gets the database actors list.
+     *
+     * @return list of {@link Actor}.
+     */
+    public List<Actor> getActorList() {
+        return actorList;
+    }
+
+    /**
+     * Adds a movie in the database movie list.
+     *
+     * @param name   movie's name.
+     * @param actors array of actor's names.
+     */
+    public void addMovie(final String name, final String[] actors) {
+        if (isNewMovie(name, movieList)) {
+            Movie newMovie = new Movie(name);
+            List<Actor> movieActors = Arrays.stream(actors).map(Actor::new)
+                    .collect(Collectors.toCollection(ArrayList::new));
```

Comment:
this may refactored like this
```java
.collect(Collectors.toList());
```

Reply:
I was using a Arraylist type instead of the generalized interface, I forgot change the collector. Done.